api/v1alpha1/virtualmachineclass_types.go

```diff
@@ -8,6 +8,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// VirtualMachineConfigSpec contains additional virtual machine
+// configuration settings including hardware specifications for the VirtualMachine
+type VirtualMachineConfigSpec struct {
+	// XML contains a vim.vm.ConfigSpec object that has been serialized to XML
+	// and base64-encoded. We only support XML for now, but that may change
+	// in the future.
```

Comment:
> We only support XML for now, but that may change in the future.

I think this makes more sense in the description for this struct, or omit it since XML is the only field defined.